### PR TITLE
Perf test: load css using RAF instead of setTimeout

### PR DESCRIPTION
**Pull request by sndrs**
I've not been able to get an iPad 2 to crash locally with this change. This should see if that's just me or it's a real improvement...

cc/ @rich-nguyen @gklopper

### CODE REVIEW

File: common/app/templates/headerInlineJS/enableStylesheets.scala.js [hunk 1]
```diff
@@ -30,6 +71,7 @@
             }
         }
         setTimeout(setMedia, null, styleSheet);
+        }
```
Comment (OliverJAsh) [inline common/app/templates/headerInlineJS/enableStylesheets.scala.js]: Indentation here?

File: common/app/templates/headerInlineJS/enableStylesheets.scala.js [hunk 2]
```diff
@@ -21,6 +21,47 @@
...
+                    styleSheet.media = "screen";
+                    return true;
+                }
+            }
+            requestAnimationFrame(function () {
+                setMedia(styleSheet);
+            });
+        }
+    } else {
     function setMedia (styleSheet) {
```
Comment (OliverJAsh) [inline common/app/templates/headerInlineJS/enableStylesheets.scala.js]: and here?

File: common/app/templates/headerInlineJS/enableStylesheets.scala.js [hunk 3]
```diff
@@ -21,6 +21,47 @@
...
+
+            if (!window.requestAnimationFrame)
+                window.requestAnimationFrame = function(callback, element) {
+                    var currTime = new Date().getTime();
+                    var timeToCall = Math.max(0, 16 - (currTime - lastTime));
+                    var id = window.setTimeout(function() { callback(currTime + timeToCall); },
+                      timeToCall);
+                    lastTime = currTime + timeToCall;
+                    return id;
+                };
```
Comment (OliverJAsh) [inline common/app/templates/headerInlineJS/enableStylesheets.scala.js]: I wonder if we should have separate beacons for native and polyfilled rAF.

Comment (sndrs) [inline common/app/templates/headerInlineJS/enableStylesheets.scala.js]: that's only a massive precaution, it's [almost not needed](http://caniuse.com/#feat=requestanimationframe) (on mobile anway)

**Comment by gklopper** [comment]
:+1: SHIPPIT

**Comment by rich-nguyen** [comment]
:+1:

**Pull request merged**